sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/implementation/ChatThreadsImpl.java

```diff
@@ -674,7 +674,7 @@ public Response<SendChatMessageResult> sendChatMessageWithResponse(
      * Gets a list of messages from a thread.
      *
      * @param chatThreadId The thread id of the message.
-     * @param maxPageSize The maximum number of messages to be returned per page. The limit can be found at https://docs.microsoft.com/en-us/azure/communication-services/concepts/service-limits#size-limits.
+     * @param maxPageSize The maximum number of messages to be returned per page.
      * @param startTime The earliest point in time to get messages up to. The timestamp should be in RFC3339 format:
```

Comment:
@jiminwen-msft did you change the description in the swagger JSON file?

Reply:
I changed json files in the spool gateway package. Not sure why it is not reflected while generating auto rest.

Reply:
You need to update here https://github.com/Azure/azure-rest-api-specs/tree/main/specification/communication/data-plane/Chat